site/src/pages/docs/server-annotated-service.mdx

```diff
@@ -187,7 +187,17 @@ one of the following supported types:
 - `String`
 - `Enum`
 - `UUID`
+- `Duration`
+- `Instant`
+- `LocalDate`
+- `LocalDateTime`
+- `LocalTime`
+- `OffsetDateTime`
+- `OffsetTime`
 - `Period`
+- `ZonedDateTime`
+- `ZoneId`
+- `ZoneOffset`
```

Comment:
Could we make this list compact? e.g.

```
- Primitive types and their wrapper types: `boolean`, `byte`, .... and `double`
- Well-known JDK classes: `String`, `Enum`, `UUID`, ... and `ZoneOffset`
```

Reply:
How about this? `Well-known JDK classes: String, Enum, UUID, ... and ZoneOffset` can be confused.

e.g:
```
- Primitive types and their wrapper types:
  - `boolean` or `Boolean`
  - `byte` or `Byte`
  - `short` or `Short`
  - `integer` or `Integer`
  - `long` or `Long`
  - `float` or `Float`
  - `double` or `Double`
- Well-known JDK classes:
  - `String`
  - `Enum`
  - `UUID`
  - `java.time` package classes
```

But, this PR does not support some `java.time` classes (`Clock`, `MonthDay`, `Year`). In this aspect, it seems better to show clearly.
```
  - `java.time` package classes
    -  `Instant`
    - `Duration` and  `Period`
    - `LocalDate`, `LocalDateTime` and `LocalTime`, 
    - `OffsetDateTime` and `OffsetTime`
    - `ZonedDateTime`, `ZoneId` and `ZoneOffset`
```

How do you think?

Reply:
The nested list with detailed list of java.time classes sounds good to me.